server/config/config.go

```diff
@@ -163,6 +163,9 @@ type ServerConfig struct {
 
 	EnableGRPCGateway bool
 
+	// ExperimentalEnabledTracing enables tracing via OpenTelemetry.
+	ExperimentalEnabledTracing bool
```

Comment:
`s/Enabled/Enable/`?

Reply:
nit: I prefer EnableExperimentalTracing to be consistent with the flag above it.

Reply:
I was following the examples of `ExperimentalMemoryMlock` and `ExperimentalTxnModeWriteWithSharedBuffer` that exist already in the `ServerConfig `.

Reply:
ah

Reply:
maybe ExperimentalTracing then?

Reply:
@gyuho suggested the distributed part as etcd already has its own tracing, see this https://github.com/etcd-io/etcd/pull/12919#issuecomment-833448429